common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -79,7 +80,7 @@ public static void setLocalDirStickyBit(String dir) {
         Runtime.getRuntime().exec("chmod +t " + dir);
       }
     } catch (IOException e) {
-      LOG.info("Can not set the sticky bit of the directory: {}", dir);
+      LOG.info("Can not set the sticky bit of the directory: {}", dir, e);
```

Comment:
I omitted it so that the stack trace would be printed. Do you think this is too verbose? I'd be equally ok with just logging `e.toString()`